README.md

```diff
@@ -638,6 +730,37 @@ session.unsubscribe();
 ```
 - This method unsubscribes from any active NewAttestation event subscriptions. It is used to stop listening for NewAttestation events when they are no longer needed.
 
+## `zkVerifySession.registerDomain`
+
+```typescript
+session.registerDomain(aggregationSize, queueSize);
+```
+- register a new domain where the owner is the signer that emits a new aggregation every aggregationSize proofs and where there could be at most  queueSize aggregation in waiting for publication state.
+* @param {number} aggregationSize - The size of the aggregation, integer equal to or less than 128.
+* @param {number} queueSize: an optional integer smaller equal than 16. 16 if it’s null.
+* @returns {{ events: EventEmitter; domainIdPromise: `Promise<number>` }}
```

Comment:
Docusaurus mdx compiler still complains.
What if it followed the format of the other sections:
```suggestion
* Returns: `{ events: EventEmitter; domainIdPromise: Promise<number> }`
```